highfidelity/hifi
# Fixing collisionless grabbing

MS#[12661](https://highfidelity.manuscript.com/f/cases/edit/12661/Entities-that-are-collisionless-are-not-grabbable-even-with-the-grabbable-property-set-to-true)

- Allowing collisionless grabbing using the mouse

**TEST PLAN**
- Launch Interface in Desktop mode
- Create a cube; ensure the properties of the cube are set to grabbable as true, and collisionless as false
- You should be able to click and drag the cube around
- Change collisionless property of the same cube to true
- You should be able to click and drag the cube around
